db/migrate/20171017142220_create_restaurants.rb

```diff
@@ -0,0 +1,20 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
+
+      t.string :name, null: false
+      t.string :address, null: false
+      t.string :city, null: false
+      t.string :state, null: false
+      t.string :zip, null: false
+      t.string :dollar_value, null: false
```

Comment:
we might best represent this value as an integer. 1 = '$', 2 = '$$', 3 = '$$$' - this will make querying and searching restaurants based on price easier.